code/datums/gamemodes/_game_mode.dm

```diff
@@ -802,13 +862,17 @@ GLOBAL_LIST_INIT(bioscan_locations, list(
 /client/proc/roundend_report_file()
 	return "data/[ckey].html"
 
-/datum/game_mode/proc/announce_xenomorphs()
+/datum/game_mode/proc/announce_memorial()
+
 	var/list/parts = list()
-	var/datum/hive_status/normal/HN = GLOB.hive_datums[XENO_HIVE_NORMAL]
-	if(!HN.living_xeno_ruler)
+
+	if(!length(GLOB.fallen_marines))
 		return
 
-	parts += span_round_body("The surviving xenomorph ruler was:<br>[HN.living_xeno_ruler.key] as [span_boldnotice("[HN.living_xeno_ruler]")]")
+	parts += "In memory of:"
+	for(var/mob/living/carbon/human/fallenmarine in GLOB.fallen_marines)
+		parts += "<b>[fallenmarine.real_name], [fallenmarine.job.title]</b>"
```

Comment:
Given that all you're using fallen_marines for is to pull strings for name and job, why not use the existing lists that the memorial/dogtags have for those two things specifically instead of keeping mob refs?